controller/auth.js

```diff
@@ -0,0 +1,80 @@
+const bcrypt = require("bcrypt");
+
+const { statusCode, message } = require("../constant/constant");
+const User = require("../model/user-model");
+
+const { success, error } = require("../response-api/responseApi");
+
+const register = async (req, res) => {
+  try {
+    const { userName, email, password } = req.body;
+
+    let user = await User.findOne({ email });
+
+    if (user) {
+      res
+        .status(statusCode.CONFLICT)
+        .json(error(message.CONFLICT), res.statusCode);
+    } else {
```

Comment:
no need to wrap the code in else block here. because you are already returning in if block.

Reply:
ok I'll remove else block